### DISCUSSION TITLE
Experience: Which antenna for HMS and CMT2300A?

### DISCUSSION THREAD

**Discussion by MichaMEG** [tbnobody/OpenDTU#2309] [Q&A]
Because the reception quality is very poor with the throw antennas (https://shop.allianceapps.io/products/allianceapps-opendtu-fusion?pr_prod_strat=e5_desc&pr_rec_id=75a10c2e4&pr_rec_pid=8454540132680&pr_ref_pid=8439160406344&pr_seq=uniform). What can you recommend here?

I ordered this one, but it hasn't arrived yet:
https://www.amazon.de/gp/aw/d/B0978Q7N7C?ref=ppx_pt2_mob_b_prod_image

After changing the position of the inverter, it is now slightly better (HMS-400-1T):
Gesendete Anfragen	698	
Empfang Erfolgreich	632	91 %
Empfang Fehler: Nichts empfangen	54	8 %
Empfang Fehler: Teilweise empfangen	0	0 %
Empfang Fehler: Beschädigt empfangen	12	2 %
Gesendete Fragment Wiederanforderungen	14	

Before that, only max 40 per cent of parcels arrive.

Thanks 

Michael

**Comment by MichaMEG** [comment]
The older Hoymiles HM-600 via NRF24 do not seem to have any problems.

**Reply by stefan123t** [reply]
HM inverters use nRF24L01+ with a common base frequency in the WIFI/Bluetooth 2.4GHz range.
Whereas HMS/HMT inverters use CMT2300A for communicating in the 868MHz frequency band in EU and 915MHz in US/NA regions.
Please take care to use an antenna adopted for the intended RF frequency range.

Maybe your “throw” antenna for the CMT2300A got mixed up ? You may ask the seller for a proper replacement as I have not heard of many complaints regarding the foil antennas the OpenDTU Fusion boards come delivered with.

**Reply by MichaMEG** [reply]
Thank you for the detailed answer.
I realise that the antennas must match the frequency.
How can I tell the difference between the “throw” antenna, different lengths?
